pkg/apis/trainer/v1alpha1/trainjob_types.go

```diff
@@ -260,6 +260,12 @@ type PodSpecOverride struct {
 	// +listType=map
 	// +listMapKey=name
 	Containers []ContainerOverride `json:"containers,omitempty"`
+
+	// SchedulingGates overrides the scheduling gates of the Pods in the target job templates.
```

Comment:
Just to keep comments consistents.
```suggestion
	// SchedulingGates overrides the scheduling gates of the Pods in the desired job templates.
```

Reply:
I've changed the other occurrences of "desired job" to "target job" as it seems more consistent with the `.spec.podSpecOverrides.targetJob` schema. Let me know what you prefer.

Reply:
Sure, thank you @astefanutti!